### PR TITLE
POM file

### LINKED ISSUE #16: Write a new interpreter
After understanding the architecture of the zeppelin-sqlflow plug-in and important code files, how to contributea new Interpreter to Zeppelin releases? You can refer to `Writing a New SQLFlow Interpreter.md`. Through a simple case, to get familiar with the creation process of an interpreter.

### PR DESCRIPTION
**Pull request by Klay1124**
fix #16

POM file for the new interpreter project.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+			</plugin>
+			<plugin>
+				<artifactId>maven-resources-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<artifactId>maven-shade-plugin</artifactId>
+			</plugin>
+		</plugins>
+	</build>
+
```
Comment (weiguoz) [inline pom.xml]: Remove the empty line.

Comment (Klay1124) [reply pom.xml]: The empty line has been deleted.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-clean-plugin</artifactId>
+					<configuration>
+						<filesets>
+							<fileset>
+								<directory>${project.build.directory}/interpreter/${interpreter.name}
+								</directory>
+								<followSymlinks>false</followSymlinks>
```
Comment (weiguoz) [inline pom.xml]: indent?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+							</fileset>
+						</filesets>
+					</configuration>
+				</plugin>
+
+				<!--<plugin>
```
Comment (weiguoz) [inline pom.xml]: Why we need these commented lines?

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+		<dependency>
+    		<groupId>com.alibaba</groupId>
+    		<artifactId>fastjson</artifactId>
+   			 <version>1.2.73</version>
+		</dependency>
+	
+	</dependencies>
+
+	<build>
+
```
Comment (weiguoz) [inline pom.xml]: Meaningless blank.

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+					<executions>
+						<execution>
+							<id>copy-interpreter-setting</id>
+							<phase>package</phase>
+							<goals>
+								<goal>resources</goal>
+							</goals>
+							<configuration>
+								<outputDirectory>${project.build.directory}/interpreter/${interpreter.name}
+								</outputDirectory>
```
Comment (weiguoz) [inline pom.xml]: one line?

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+		<junit.version>4.12</junit.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.zeppelin</groupId>
+			<artifactId>zeppelin-interpreter-shaded</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+
```
Comment (weiguoz) [inline pom.xml]: Meaningless blank.

File: pom.xml [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+  			<artifactId>jsqlflow</artifactId>
+  			<version>0.2.3</version>
+		</dependency>
+		
+		<dependency>
+    		<groupId>com.alibaba</groupId>
+    		<artifactId>fastjson</artifactId>
+   			 <version>1.2.73</version>
+		</dependency>
+
```
Comment (weiguoz) [inline pom.xml]: Meaningless blank.

File: pom.xml [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+					<configuration>
+						<filters>
+							<filter>
+								<artifact>*:*</artifact>
+								<excludes>
+									<exclude>META-INF/*.SF</exclude>
+									<exclude>META-INF/*.DSA</exclude>
+									<exclude>META-INF/*.RSA</exclude>
+									<exclude>org.apache.zeppelin:zeppelin-interpreter-shaded
+									</exclude>
```
Comment (weiguoz) [inline pom.xml]: one line?

File: pom.xml [hunk 9]
```diff
@@ -0,0 +1,202 @@
...
+								implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
+							<transformer
+								implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+								<resource>reference.conf</resource>
+							</transformer>
+						</transformers>
+						<artifactSet>
+							<excludes>
+								<exclude>org.apache.zeppelin:zeppelin-interpreter-shaded
+								</exclude>
```
Comment (weiguoz) [inline pom.xml]: one line?

**Review by weiguoz — CHANGES_REQUESTED** [review]
Please review the file carefully, thanks.

File: pom.xml (hunk 2 above)
Comment (Klay1124) [reply pom.xml]: It has been corrected.

File: pom.xml (hunk 3 above)
Comment (Klay1124) [reply pom.xml]: It has been corrected.

File: pom.xml (hunk 4 above)
Comment (Klay1124) [reply pom.xml]: It has been corrected.

File: pom.xml (hunk 5 above)
Comment (Klay1124) [reply pom.xml]: It has been corrected.

File: pom.xml (hunk 6 above)
Comment (Klay1124) [reply pom.xml]: It has been corrected.

File: pom.xml (hunk 7 above)
Comment (Klay1124) [reply pom.xml]: It has been corrected.

File: pom.xml (hunk 8 above)
Comment (Klay1124) [reply pom.xml]: It has been corrected.

File: pom.xml (hunk 9 above)
Comment (Klay1124) [reply pom.xml]: It has been corrected.

**Review by weiguoz — APPROVED** [review]

**Pull request merged**